nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -351,6 +351,17 @@ let
           ''}
         '';
 
+        accessLogString = { file, format, buffer, gzip, flush, off ? [], ... }: "
+          access_log "
+            + optionalString (off == false && file != null) "${file} "
```

Comment:
Better do a `if/else` based on the value of `off`, it makes the code more readable.

Reply:
Fixed.